Directory.Build.props

```diff
@@ -35,8 +35,8 @@
          * Major/Minor match Android stable API level, such as 30.0 for API 30.
          * Bump first digit of the patch version for feature releases (and reset the first two digits to 0)
     -->
-    <AndroidPackVersion>35.0.0</AndroidPackVersion>
-    <AndroidPackVersionSuffix>rc.2</AndroidPackVersionSuffix>
+    <AndroidPackVersion>35.0.1</AndroidPackVersion>
+    <AndroidPackVersionSuffix>rtm</AndroidPackVersionSuffix>
     <IsStableBuild>false</IsStableBuild>
     <IsStableBuild Condition=" '$(AndroidPackVersionSuffix)' == 'rtm' ">true</IsStableBuild>
```

Comment:
I think this condition may be problematic with these changes -- we're adding rtm label but not bringing in other changes which make the package version stable? We may want to move this in to `build-tools/scripts/XAVersionInfo.targets` now, and update the condition to:

```xml
 <IsStableBuild Condition=" '$(AndroidPackVersionSuffix)' == 'rtm' and $(XAVersionBranch.StartsWith('release/')) ">true</IsStableBuild>
```